internal/annotators/http/pki.go

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright 2021 Dell Inc.
```

Comment:
Change year in copyright to 2022. Same for any other new files.

Reply:
Fixed in ef52a4d46b344e24703744e982ac2fc17aed75c4